contracts/interfaces/IDelegation.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity 0.5.16;
+
+import "./IContractRegistry.sol";
+
+/// @title Elections contract interface
+interface IDelegations {
+    // Delegation state change events
+	event StakeChanged(address addr, uint256 stake);
+    event DelegatedStakeChanged(address addr, uint256 selfSstake, uint256 delegatedStake);
+
+    // Function calls
+	event Delegated(address from, address to, );
+
+	/*
+     * External methods
+     */
+
+	/// @dev Stake delegation
+	function delegate(address to) external;
+    
+	/*
+     * Methods restricted to other Orbs contracts
+     */
+
+	/// @dev Called by: delegation contract
+	/// Notifies a stake change event
+	function stakeChange(address addr, uint256 selfStake, uint256 delegatedStake); /* onlyDelegationContract */;
```

Comment:
But this is the delegations contract, was this supposed to be somewhere else, or removed?

Reply:
Fixed